.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,107 @@
+# **what?**
+# This workflow will take the new version number to bump to. With that
+# it will run versionbump to update the version number everywhere in the
+# code base and then run changie to create the corresponding changelog.
+# A PR will be created with the changes that can be reviewed before committing.
+
+# **why?**
+# This is to aid in releasing dbt-trino and making sure we have updated
+# the version in all places and generated the changelog.
+
+# **when?**
+# This is triggered manually
+
+name: Version Bump
```

Comment:
This is so cool, it would be nice to have something similar for bumping Trino and Starburst versions.